pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Param.cpp

```diff
@@ -248,7 +247,7 @@ TEMPLATE_TEST_CASE("StateVectorKokkosManaged::applyRZ",
 
 TEMPLATE_TEST_CASE("StateVectorKokkosManaged::applyPhaseShift",
                    "[StateVectorKokkosManaged_Param]", double) {
-    const bool inverse = GENERATE(true, false);
+    const bool inverse = GENERATE(false, true);
```

Comment:
Any reason for the order swap?

Reply:
Just seemed more natural to iterate from 0 to 1 (casting in my head), or start with the default inverse value.